src/locales/en_US.ts

```diff
@@ -109,10 +109,10 @@ const strings = {
   warning_please_wait_title: 'Please Wait',
 
   warning_scam_title: 'Scam Warning',
-  warning_scam_message_financial_advice: 'Edge will not give financial advice.',
+  warning_scam_message_financial_advice_s: '%s will not give financial advice.',
   warning_scam_message_irreversibility: 'Cryptocurrency transactions are irreversible.',
   warning_scam_message_unknown_recipients: 'Do not send money to people or organizations you do not know.',
-  warning_scam_footer: 'If you have any questions or concerns regarding this send please contact support@edge.app',
+  warning_scam_footer_s: 'If you have any questions or concerns regarding this send please contact %s',
```

Comment:
`If you have any questions or concerns regarding this send, please contact %s`

Reply:
fixed